test/test_core/test_integration.py

```diff
@@ -93,14 +93,18 @@ def test_kriging_mutation(interpolator_islith_isfault, map_sequential_pile):
     #plt.savefig(os.path.dirname(__file__)+'/figs/test_kriging_mutation')
 
     geo_model.modify_kriging_parameters('range', 1)
-
     geo_model.modify_kriging_parameters('drift equations', [0, 3])
+
     print(geo_model.solutions.lith_block, geo_model.additional_data)
+    # copy dataframe before interpolator is calculated
+    pre = geo_model.additional_data.kriging_data.df.copy()
 
```

Comment:
Hey @javoha! I was wondering what this part of the pull request is about?

Reply:
Ah, I just saw the commit form a couple of weeks ago on "When using modify_kriging_parameters and recompiling the interpolator, the changes are not taken into account and reset to default." - Thanks!

Reply:
Actually I was also wondering - Miguel merged that piece of code, I guess and I kept my remote branch up to date. Still in this pull request it was included - can you tell me why?

Reply:
Well, apparently it's not part of the master branch - but I can't tell you why 😄 Some merging accident maybe?

Reply:
Mmh I think @Leguark wanted to change it a little. Maybe this should not be in this pull request then.

Reply:
yes there is a merge conflict, I will try to deal with it as soon as my enviroment is ready